networking/gcp/vpc/variables.tf

```
@@ -0,0 +1,61 @@
+variable "project" {
+  description = "Project name"
+  type        = string
+}
+
+variable "region" {
+  description = "Region where to deploy the different subnets"
+  type        = string
+}
+
+variable "name" {
+  description = "Name of the VPC"
+  type        = string
+  default     = "armonik-vpc"
+}
+
+variable "private_subnets" {
+  description = "A list of private subnets inside the VPC"
+  type        = list(string)
```

```suggestion
  type        = set(string)
```

It is better to use sets. this allows more easily to add and remove elements in following applies.